src/components/dashboard/dashboard.tmpl.html

```diff
@@ -27,10 +27,9 @@
 
     <!-- Displayed to MYOB Essentials Users -->
     <div id="myob-message" ng-if="hasMyobEssentialsOnly && myobMessageConfig.show" class="alert alert-warning">
-      <h3>It looks like you are using MYOB Essentials...</h3>
-      <p>Unfortunately, all the information required by Impac! is not exposed by MYOB Essentials API <strong>yet</strong>.
-      Consequently, most of the accounting widgets provided by Impac! won't find a valid source of data to display their content.</p>
-      <p>Maybe you should consider switching to <a href="http://myob.com.au/products/small-business/accounting-software/myob-accountright-1422735752948" target="_blank">MYOB AccountRight</a>, the other MYOB accounting solution?</p>
+      <h3 translate>impac.dashboard.it_looks_like</h3>
+      <p translate="impac.dashboard.unfortunately_information"></p>
```

Comment:
Yes, it is correct. It is for compiling string to html if this string is contain some html tags. In this case: 
"impac.dashboard.unfortunately_information": "Unfortunately, all the information required by Impac! is not exposed by MYOB Essentials API <strong>yet</strong>......."